src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Dynamo.Core;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.Wpf.Authentication;
+using Greg.AuthProviders;
+using Reach;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Dynamo.Publish.Models
+{
+    public class PublishModel
+    {
+        private readonly AuthenticationManager manager;
+        public LoginService LoginService;
```

Comment:
Let's remove `LoginService` here.  Other parts of the application will handle ensuring the `AuthProvider.RequestLogin` causes the creation of